prime-angular-frontend/src/app/modules/phsa-labtech/phsa-labtech-routing.module.ts

```diff
@@ -3,23 +3,25 @@ import { Routes, RouterModule } from '@angular/router';
 
 import { PhsaLabtechRoutes } from './phsa-labtech.routes';
 import { PhsaLabtechDashboardComponent } from './shared/components/phsa-labtech-dashboard/phsa-labtech-dashboard.component';
+import { CanDeactivateFormGuard } from '@core/guards/can-deactivate-form.guard';
 
-import { ExampleComponent } from './pages/example/example.component';
+import { AccessCodeComponent } from './pages/access-code/access-code.component';
 
 const routes: Routes = [
   {
     path: '',
     component: PhsaLabtechDashboardComponent,
     children: [
-      {
-        path: PhsaLabtechRoutes.EXAMPLE,
-        component: ExampleComponent,
-        data: { title: 'Example' }
-      },
       {
```

Comment:
It doesn't affect routing, but we tend to keep the redirect route at the bottom of the route configuration since it is mainly used after a user has encountered an error, and we want a base module route that will invoke the module guards and/or take them where they need to go.

Reply:
moved redirect route to bottom